[2024-09-02 12:11:35 +0000] [93848] [INFO] Starting gunicorn 23.0.0
[2024-09-02 12:11:35 +0000] [93848] [INFO] Listening at: http://10.178.0.2:8080 (93848)
[2024-09-02 12:11:35 +0000] [93848] [INFO] Using worker: sync
[2024-09-02 12:11:35 +0000] [93849] [INFO] Booting worker with pid: 93849
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '34.64.196.23'. You may need to add '34.64.196.23' to ALLOWED_HOSTS.
Bad Request: /login
[2024-09-02 12:19:36 +0000] [93848] [INFO] Handling signal: term
[2024-09-02 21:19:36 +0900] [93849] [INFO] Worker exiting (pid: 93849)
[2024-09-02 12:19:36 +0000] [93848] [INFO] Shutting down: Master
[2024-09-02 12:22:39 +0000] [93928] [INFO] Starting gunicorn 23.0.0
[2024-09-02 12:22:39 +0000] [93928] [INFO] Listening at: http://10.178.0.2:8080 (93928)
[2024-09-02 12:22:39 +0000] [93928] [INFO] Using worker: sync
[2024-09-02 12:22:39 +0000] [93929] [INFO] Booting worker with pid: 93929
Not Found: /api/api/preregister
Not Found: /api/preregister
[2024-09-02 13:26:15 +0000] [93928] [ERROR] Worker (pid:93929) was sent SIGHUP!
[2024-09-02 13:26:15 +0000] [101024] [INFO] Booting worker with pid: 101024
[2024-09-02 13:26:15 +0000] [93928] [INFO] Handling signal: hup
[2024-09-02 13:26:15 +0000] [93928] [INFO] Hang up: Master
[2024-09-02 13:26:15 +0000] [93928] [INFO] Handling signal: term
[2024-09-02 13:26:15 +0000] [101027] [INFO] Booting worker with pid: 101027
[2024-09-02 22:26:15 +0900] [101024] [INFO] Worker exiting (pid: 101024)
[2024-09-02 13:26:45 +0000] [93928] [INFO] Shutting down: Master
[2024-09-02 13:27:50 +0000] [878] [INFO] Starting gunicorn 23.0.0
[2024-09-02 13:27:50 +0000] [878] [INFO] Listening at: http://10.178.0.2:8080 (878)
[2024-09-02 13:27:50 +0000] [878] [INFO] Using worker: sync
[2024-09-02 13:27:50 +0000] [879] [INFO] Booting worker with pid: 879
Watching for file changes with StatReloader
[04/Sep/2024 10:52:33] "GET /preregistration_list/?date=all&usage=all HTTP/1.0" 200 8432
[04/Sep/2024 10:52:34] "GET /preregistration_list/?date=all&usage=all HTTP/1.0" 200 8432
[04/Sep/2024 10:52:41] "GET /login/ HTTP/1.0" 200 1871
[04/Sep/2024 10:52:47] "POST /login/ HTTP/1.0" 302 0
[04/Sep/2024 10:52:47] "GET /preregistration_list/ HTTP/1.0" 200 8432
[04/Sep/2024 10:53:13] "GET /login/ HTTP/1.0" 200 1871
[04/Sep/2024 10:55:45] "GET /login/ HTTP/1.0" 200 1871
[04/Sep/2024 10:55:51] "POST /login/ HTTP/1.0" 302 0
[04/Sep/2024 10:55:51] "GET /preregistration_list/ HTTP/1.0" 200 9115
[04/Sep/2024 10:55:53] "GET /preregistration_list/?date=2024-09-04 HTTP/1.0" 200 5105
[04/Sep/2024 10:55:55] "GET /preregistration_list/?date=2024-09-03 HTTP/1.0" 200 8616
[04/Sep/2024 10:55:55] "GET /preregistration_list/?date=2024-09-04 HTTP/1.0" 200 5105
[04/Sep/2024 10:55:56] "GET /preregistration_list/?date=all HTTP/1.0" 200 9115
[04/Sep/2024 10:56:02] "GET /preregistration_list/?date=all&usage=used HTTP/1.0" 200 7164
[04/Sep/2024 10:56:02] "GET /preregistration_list/?date=all&usage=unused HTTP/1.0" 200 6515
[04/Sep/2024 10:56:04] "GET /preregistration_list/?date=all&usage=used HTTP/1.0" 200 7164
[04/Sep/2024 10:56:04] "GET /preregistration_list/?date=all&usage=all HTTP/1.0" 200 9115
[04/Sep/2024 11:08:18] "GET /preregistration_list/?date=all&usage=all HTTP/1.0" 200 9150
[04/Sep/2024 11:10:15] "GET /preregistration_list/?date=all&usage=all HTTP/1.0" 200 9150
[04/Sep/2024 11:11:11] "GET /preregistration_list/?date=all&usage=all HTTP/1.0" 200 10678
[04/Sep/2024 11:11:12] "GET /preregistration_list/?date=2024-09-03 HTTP/1.0" 200 8651
[04/Sep/2024 11:11:13] "GET /preregistration_list/?date=2024-09-04 HTTP/1.0" 200 6632
[04/Sep/2024 11:11:13] "GET /preregistration_list/?date=all HTTP/1.0" 200 10678
[04/Sep/2024 11:17:09] "GET /login/ HTTP/1.0" 200 1871
[04/Sep/2024 11:26:48] "GET /preregistration_list/?date=all HTTP/1.0" 200 10678
[04/Sep/2024 11:26:59] "GET /preregistration_list/?date=all HTTP/1.0" 200 10678
[04/Sep/2024 11:27:00] "GET /preregistration_list/?date=2024-09-04 HTTP/1.0" 200 6632
[04/Sep/2024 11:27:01] "GET /preregistration_list/?date=2024-09-03 HTTP/1.0" 200 8651
[04/Sep/2024 11:27:03] "GET /preregistration_list/?date=all HTTP/1.0" 200 10678
[04/Sep/2024 11:27:04] "GET /preregistration_list/?date=all&usage=all HTTP/1.0" 200 10678
[04/Sep/2024 11:27:04] "GET /preregistration_list/?date=all&usage=used HTTP/1.0" 200 8735
[04/Sep/2024 11:27:06] "GET /preregistration_list/?date=all&usage=unused HTTP/1.0" 200 6506
[04/Sep/2024 11:27:07] "GET /preregistration_list/?date=all&usage=used HTTP/1.0" 200 8735
[04/Sep/2024 11:27:09] "GET /preregistration_list/?date=all&usage=unused HTTP/1.0" 200 6506
[04/Sep/2024 11:27:11] "GET /preregistration_list/?date=all&usage=used HTTP/1.0" 200 8735
[04/Sep/2024 11:27:11] "GET /preregistration_list/?date=all&usage=unused HTTP/1.0" 200 6506
[04/Sep/2024 11:27:13] "GET /preregistration_list/?date=2024-09-03 HTTP/1.0" 200 8651
[04/Sep/2024 11:27:15] "GET /preregistration_list/?date=2024-09-03&usage=unused HTTP/1.0" 200 6042
[04/Sep/2024 11:27:19] "GET /preregistration_list/?date=2024-09-04 HTTP/1.0" 200 6632
[04/Sep/2024 11:27:20] "GET /preregistration_list/?date=2024-09-04&usage=unused HTTP/1.0" 200 5070
[04/Sep/2024 11:27:21] "GET /preregistration_list/?date=all HTTP/1.0" 200 10678
[04/Sep/2024 11:27:22] "GET /preregistration_list/?date=all&usage=unused HTTP/1.0" 200 6506
[04/Sep/2024 11:27:23] "GET /preregistration_list/?date=2024-09-03 HTTP/1.0" 200 8651
[04/Sep/2024 11:27:24] "GET /preregistration_list/?date=2024-09-03&usage=unused HTTP/1.0" 200 6042
[04/Sep/2024 11:27:34] "GET /preregistration_list/?date=2024-09-03&usage=unused HTTP/1.0" 200 6042
[04/Sep/2024 11:27:35] "GET /preregistration_list/?date=2024-09-03&usage=all HTTP/1.0" 200 8651
[04/Sep/2024 11:27:36] "GET /preregistration_list/?date=all HTTP/1.0" 200 10678
[04/Sep/2024 11:27:37] "GET /preregistration_list/?date=all&usage=unused HTTP/1.0" 200 6506
[04/Sep/2024 11:27:39] "GET /preregistration_list/?date=2024-09-03 HTTP/1.0" 200 8651
[04/Sep/2024 11:27:41] "GET /preregistration_list/?date=2024-09-03&usage=used HTTP/1.0" 200 7194
[04/Sep/2024 11:27:46] "GET /preregistration_list/?date=2024-09-03&usage=unused HTTP/1.0" 200 6042
[04/Sep/2024 11:27:47] "GET /preregistration_list/?date=all HTTP/1.0" 200 10678
[04/Sep/2024 11:27:56] "GET /preregistration_list/?date=2024-09-04 HTTP/1.0" 200 6632
[04/Sep/2024 11:27:56] "GET /preregistration_list/?date=2024-09-03 HTTP/1.0" 200 8651
[04/Sep/2024 11:27:57] "GET /preregistration_list/?date=all HTTP/1.0" 200 10678
